### ISSUE TITLE
investigate navbar accessibility

### ISSUE THREAD

**Issue by johrstrom** [OSC/ondemand#945]
In the bootstrap 4 upgrade it became apparent that we don't really use roles in the navbar, though a lot of WCAG documentation seems to indicate there are several.

This ticket is to investigate if our navbar currently exhibits expected navigation patterns and announcements and propose (or supply) fixes if required.

Here are some docs on expected behaviour and an example.

https://www.w3.org/WAI/GL/wiki/Using_ARIA_menus
https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201735133575781/1201737596540914) by [Unito](https://www.unito.io)

**Comment by johrstrom** [comment]
The landmark seems funny. How it is read in a screen reader is below.  The ERB line being correct, but it seems like there are 1-2 extra things being read here.

```
banner landmark
navigation
naviation landmark
<%= title %> link
```

**Comment by johrstrom** [comment]
This was fixed in #3299.
